### DISCUSSION TITLE
Show input text inside a div?

### DISCUSSION THREAD

**Discussion by cipriano200** [unpoly/unpoly#546] [Q&A] [answered]
Does Unpoly has an API to put value from an input inside a div? Or any other solution?

```
<input type="text" name="name">
        
<div class="name"></div>
```

**Comment by alexandremjacques** [comment, marked as answer]
If you're trying to achieve something like a reactive behavior from Vue or React, I guess there's nothing direct related.

Though, you could achieve what you want with vanilla javacript and Unpoly `up.on` (https://unpoly.com/up.on):

```html
<p>
  <input type="text" id="myinput">
</p>
    
<div id="mydiv"></div>
```

```js
up.on('keydown', '#myinput', function (event, element) {
  document.getElementById('mydiv').innerText = event.target.value;
})
```

**Reply by cipriano200** [reply]
Thank you
